### ISSUE TITLE
Empty time series if timelines of different cubes are not overlapping

### ISSUE THREAD

**Issue by AliceBalfanz** [dcs4cop/xcube-viewer#215]
When choosing a new cube, the old time series is empty, if the time frame of the new cube is not overlapping with the former cube.

**Comment by forman** [comment]
Yes, because the newly selected cube provides the new context - including it's time frame.

But we could automatically expand the time axis to cover all contributing data sources.
I created for this: https://github.com/dcs4cop/xcube-viewer/issues/284
